applications/DEMApplication/custom_constitutive/DEM_rolling_friction_model.cpp

```diff
@@ -1,8 +1,10 @@
-/////////////////////////////////////////////////
-// Main author: Chengshun Shang (CIMNE)
-// Email: chengshun.shang1996@gmail.com
-// Date: Aug 2022
-/////////////////////////////////////////////////
+//  Kratos Multi-Physics - ThermalDEM Application
```

Comment:
Hi @rlrangel. Please check the modified header, I think  it could be better if we modify `//  Kratos Multi-Physics - ThermalDEM Application` to `//  Kratos Multi-Physics-DEMApplication`. Because we are in DEMApplication now.

Reply:
That's true, I forgot to change the application name. I will do it now.